scripts/stale.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 The HuggingFace Team, the AllenNLP library authors. All rights reserved.
```

Comment:
kinda sad that this is written in Python when the GitHub Actions SDK is pure Typescript ;)

Reply:
My brain only thinks in Python now :) What are even types?